std-bits/table/src/main/java/org/enso/table/write/ExcelWriter.java

```diff
@@ -302,23 +346,25 @@ private static void writeValueToCell(Cell cell, int j, Storage storage, Workbook
    * @param topRow top row index (1-based) of the range to check.
```

Comment:
The comment above (I cannot select it in review...) is also outdated - it says it defaults to `True` but it's not returning a boolean anymore.